### ISSUE TITLE
Can't get file to upload to folder list

### ISSUE THREAD

**Issue by ColorWorksGirl** [googlecolab/colabtools#3690]
I am trying to follow the instructions on this website and from his youtube video.
https://kevinstratvert.com/2023/01/19/best-free-speech-to-text-ai-whisper-ai/#install-google-colaboratory
https://youtu.be/8SQV-B83tPU
I have dragged and dropped both a mp4 and a mp3 file (at separate times) and they both have the circle go around and then it stops and turns red. 
It says there is 84 GB available and the largest file is under 400 MB.
The file is not added to the Folder list, even if I refresh the folder list.
I have tried it both with the GPU and with None selected in the Runtime menu.
Could you please tell me what I'm doing wrong?
Thank you.
Angel

**Comment by mayankmalik-colab** [comment]
Can you try using the below code:

```
from google.colab import files

uploaded = files.upload()

for fn in uploaded.keys():
  print('User uploaded file "{name}" with length {length} bytes'.format(
      name=fn, length=len(uploaded[fn])))
```
It would be saved at  `/content`

Alternatively, you can also save your files on google drive and mount your Google Drive on your runtime. For more details you can check [example notebook](https://colab.research.google.com/notebooks/io.ipynb).

**Comment by BlockSats** [comment]
You should try this ;-)
https://github.com/ArthurFDLR/whisper-youtube

**Comment by ColorWorksGirl** [comment]
Hi BlockSats,
Thank you for that link. :)

However, when I see code like that, my eyes glaze over and my brain shuts down.

I did find an alternative solution though.

I used Movie Maker - Video Editor to extract the audio, then Microsoft Word to use that audio file to make a transcript. It wasn't perfect, but it was pretty good.

Recently I also had videos up in Dropbox and when I viewed them in Dropbox instead of downloading them, Dropbox had a closed caption option on the video and when I clicked on it, it added closed captions to the video. Not sure if they are attached and will download with the video but it worked really well while I watched them in the cloud.

Thank you for your help
ColorWorksGirl
